src/xxd/xxd.c

```diff
@@ -482,6 +501,51 @@ static unsigned char etoa64[] =
     0070,0071,0372,0373,0374,0375,0376,0377
 };
 
+   static char
+begin_coloring_char(char *l,int *c,int e,int ebcdic) {
```

Comment:
The spacing is still wonky (e.g. no space after commas).
The function's return type is `char` but it doesn't return anything. Perhaps make it return `int` so that the index `c` can be taken by value.

Reply:
I planned to use return value for something but then figured that passing pointer would be cleaner. (And then forgot type of return value.)